client/src/main/java/org/apache/rocketmq/client/ClientConfig.java

```diff
@@ -173,6 +176,7 @@ public void resetClientConfig(final ClientConfig cc) {
         this.unitName = cc.unitName;
         this.vipChannelEnabled = cc.vipChannelEnabled;
         this.useTLS = cc.useTLS;
+        this.sockProxyJson = cc.sockProxyJson;
```

Comment:
SOCKS is the abbreviation of SOCKet Secure, so it is `socksProxyJson` rather than `sockProxyJson`  here.

Reply:
+1

Reply:
I rename it to socksProxyConfig